docs/restore-individual-tables.md

```diff
@@ -1,76 +1,101 @@
 
-```text
-CREATE TABLE export_test (
+When moving tables, you'll work with these types of files:
+
+| File                             | Description |
+|----------------------------------|-------------|
+| ibdata1                          | This is the system tablespace file in InnoDB. It contains critical metadata and potentially data for various tables. It's often required for any InnoDB restore, including single table restores. |
+| undo_001 and undo_002            | These are the undo log files. They are necessary for transaction rollback and crash recovery during the prepare phase. They are required for a consistent backup. |
+| mysql.ibd                        | This file contains the InnoDB data dictionary. It's crucial for understanding the structure of your tables and is essential for restoring individual tables. |
+| xtrabackup_info                  | This file contains metadata about the backup itself, such as the server version and backup time. |
+| xtrabackup_checkpoints           | This file records checkpoint information, which is used for crash recovery and redo log application. |
+| xtrabackup_logfile               | This is the redo log file, which records all changes made to the database. It's essential for bringing the backup to a consistent state. |
+| xtrabackup_tablespaces           | The list of tablspaces that are stored external to the datadir. On restore, xtrabacup will try to restore the files to same locations as recorded in the file. |
+| .ibd from all tables you want to restore | This is the most important part for a single table restore. Each table in InnoDB has its own .ibd file, which contains the table's data and indexes. To restore a single table, you must have the .ibd file corresponding to that specific table. If you want to restore multiple tables, you need the .ibd files for each of those tables. |
+
+Here's how to export and import a table called `export_test`:
+
+```{.bash data-prompt="mysql>"}
+mysql> CREATE TABLE export_test (
 a int(11) DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 ```
 
+## Save the schema
+
+Before importing a table, you must have the schema for all tables you intend to restore. XtraBackup does not back up the table definitions, only the data files. If the original schema is lost, MySQL will not recognize the restored .ibd files.
+
+When you take a backup, you might not know which tables are included. It’s important to save the schema for all tables so you can restore them later without issues.The schema can change between the time you take the backup and when you run mysqldump, so make sure to capture the latest structure. 
+
+You could also use the script in the [Dump schema for all tables](dump-schema.md). You must modify the script for your specific use case and environment.
```

Comment:
If you used the script (dump-schema.md) during the backup, you saved the table definitions. You may use the appropriate table .SQL file to create the table.